documentation/source/quickstart.rst

```diff
@@ -18,104 +18,92 @@ Cocotb has the following requirements:
+Installing a package manager really helps things out here.
 
-Debian/Ubuntu-based
-^^^^^^^^^^^^^^^^^^^
+`Brew <https://brew.sh/>`_ seems to be the most popular, so we'll assume you have that installed.
 
 .. code-block:: bash
 
-    sudo apt-get install git make gcc g++ swig python-dev
+    brew install python icarus-verilog gtkwave
 
-Red Hat-based
-^^^^^^^^^^^^^
+.. seealso::
 
-.. code-block:: bash
+   For more installation options (also for 32-bit simulators) please see `our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
 
-    sudo yum install gcc gcc-c++ libstdc++-devel swig python-devel
 
+Simulator Installation
+^^^^^^^^^^^^^^^^^^^^^^
 
-Windows Installation
---------------------
+For detailed instructions and the list of supported simulators see :ref:`simulator-support`.
 
-Download the MinGW installer from https://osdn.net/projects/mingw/releases/.
-
-Run the GUI installer and specify a directory you would like the environment
-installed in. The installer will retrieve a list of possible packages, when this
-is done press "Continue". The MinGW Installation Manager is then launched.
+.. note::
 
-The following packages need selecting by checking the tick box and selecting
-"Mark for installation"
+    Ensure that path to the simulator executable is appended to the ``PATH`` environment variable.
 
-.. code-block:: bash
 
-    Basic Installation
-      -- mingw-developer-tools
-      -- mingw32-base
-      -- mingw32-gcc-g++
-      -- msys-base
+.. _installation_via_pip::
 
-From the Installation menu then select "Apply Changes", in the next dialog
-select "Apply".
+Cocotb Installation via PIP
+---------------------------
 
-When installed a shell can be opened using the :file:`msys.bat` file located under
-the :file:`<install_dir>/msys/1.0/`
+.. versionadded:: 1.2
 
-Python can be downloaded from https://www.python.org/downloads/windows/.
-Run the installer and download to your chosen location.
+Cocotb can be installed by running
 
-It is beneficial to add the path to Python to the Windows system ``PATH`` variable
-so it can be used easily from inside Msys.
+.. code-block:: bash
 
-Once inside the Msys shell commands as given here will work as expected.
+    pip install cocotb  # use `pip3` in case multiple of Python/pip version installed
 
-macOS Packages
---------------
+.. seealso::
 
-You need a few packages installed to get cocotb running on macOS.
-Installing a package manager really helps things out here.
+    For user local installation follow the `pip User Guide <https://pip.pypa.io/en/stable/user_guide/#user-installs/>`_.
 
-`Brew <https://brew.sh/>`_ seems to be the most popular, so we'll assume you have that installed.
+To install the development version of cocotb:
 
 .. code-block:: bash
 
-    brew install python icarus-verilog gtkwave
+    pip install https://github.com/cocotb/cocotb/archive/master.zip  # use `pip3` in case multiple of Python/pip version installed
```

Comment:
```suggestion
    pip install https://github.com/cocotb/cocotb/archive/master.zip  # use `pip3` in case multiple versions of Python or pip are installed
```